### PR TITLE
AWSLambda sample to netcoreapp3.1

### PR DESCRIPTION
**Pull request by DavidBoike**
@boblangley Take a look and let me know what you think.

### CODE REVIEW

File: samples/aws/lambda/sqs/SQSLambda_0.1/AwsLambda.SQSTrigger/Function.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public Function()
     #region FunctionHandler
     public async Task FunctionHandler(SQSEvent evnt, ILambdaContext context)
     {
-        using (var cancellationTokenSource = new CancellationTokenSource(context.RemainingTime.Subtract(DefaultRemainingTimeGracePeriod)))
```
Comment (danielmarbach) [inline samples/aws/lambda/sqs/SQSLambda_0.1/AwsLambda.SQSTrigger/Function.cs:20]: @DavidBoike any reasons this was removed? When we created the sample we deliberately added this to show how the execution can be canceled with alignment with the lambda context

Comment (danielmarbach) [reply samples/aws/lambda/sqs/SQSLambda_0.1/AwsLambda.SQSTrigger/Function.cs:20]: FYI I'm not strictly opposed to remove this. We had a debate whether we are violating the SRP of a sample or not. At the end we added it. I'm more curious to hear if it was intentional and if yes why

Comment (DavidBoike) [reply samples/aws/lambda/sqs/SQSLambda_0.1/AwsLambda.SQSTrigger/Function.cs:20]: That was totally my bad. I did that to try to get the web app API to work, and forgot to revert that before committing.

**Comment by DavidBoike** [comment]
Since you both looked at it and I fixed the one goof you both identified, I'll go ahead and merge rather than force you to re-review. Can always change it further later.

**Pull request merged**